.drone.star

```diff
@@ -225,8 +226,13 @@ def gui_test_pipeline(ctx, trigger = {}, filterTags = [], server_version = oc10_
         ],
     }]
 
-def build_client(c_compiler, cxx_compiler, build_type, generator, build_command, image = OC_CI_CLIENT):
-    cmake_options = '-G"%s" -DCMAKE_C_COMPILER="%s" -DCMAKE_CXX_COMPILER="%s" -DCMAKE_BUILD_TYPE="%s" -DBUILD_TESTING=1 -DWITH_LIBCLOUDPROVIDERS=ON' % (generator, c_compiler, cxx_compiler, build_type)
+def build_client(c_compiler, cxx_compiler, build_type, generator, build_command, image = OC_CI_CLIENT, ctest = True):
+    cmake_options = '-G"%s" -DCMAKE_C_COMPILER="%s" -DCMAKE_CXX_COMPILER="%s" -DCMAKE_BUILD_TYPE="%s" -DWITH_LIBCLOUDPROVIDERS=ON' % (generator, c_compiler, cxx_compiler, build_type)
+
+    if ctest:
+        cmake_options += " -DBUILD_TESTING=1"
+    else:
+        cmake_options += " -DBUILD_TESTING=0"
```

Comment:
not building ctest files make the build process a bit faster. And I think those test files are not required to build while testing the GUI

CC @TheOneRing

Reply:
Sure, good idea.
Nitpick: please use ON OFF.

Reply:
Right, as long as we don't run the tests anyway, there is no reason to build them. They're tested on GitHub actions.